### PR TITLE
ARM: configs: tsimx6ul: Set i2c gpio drivers to =m

### PR DESCRIPTION
**Pull request by markfeathers**
Set this to a module to address some platforms loading i2c gpio controllers before fpga gpio I/O which makes the numbering not match the documentation or older kernels.

### CODE REVIEW

**Comment by markfeathers** [comment]
I don't expect this to change so this can be reviewed now, but don't click merge once it meets the approval number since I want to add a regression test for the bug this caused.

**Review by ts-kris — APPROVED** [review]
Reviewed, not tested.

Curious to see if this helps with the issue and is verifiable via testing.

File: arch/arm/configs/tsimx6ul_defconfig
Comment (randytsi369) [inline arch/arm/configs/tsimx6ul_defconfig:1]: what are we aiming to resolve by loading these as modules?

**Review by luckshot — APPROVED** [review]

**Review by randytsi369 — APPROVED** [review]
Reviewed changes, not tested, and not sure exactly why the decision to load as modules, but should be okay.

File: arch/arm/configs/tsimx6ul_defconfig
Comment (markfeathers) [reply arch/arm/configs/tsimx6ul_defconfig:1]: See #32

**Pull request merged**